tests/TestRunner.xUnit/XUnitFilterType.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Xamarin.Android.UnitTests.XUnit
+{
+	public enum XUnitFilterType
+	{
+		Trait,
+		TypeName
```

Comment:
Please use trailing `,` on all enum/etc. values, to reduce "diff noise" when we add new enum values.